### PR TITLE
Add normalization on import

### PR DESCRIPTION
**Pull request by smn**
Currently normalizes for str, int, float and msisdns for countries we
are currently active in.

### CODE REVIEW

File: go/contacts/parsers/base.py [hunk 1]
```diff
@@ -1,11 +1,74 @@
...
+            ('', 'Leave as is'),
+            ('string', 'Plain Text'),
+            ('integer', 'Whole Numbers (0, 10, 245, ... )'),
+            ('float', 'Numbers (0.1, 3.14, 4.165, ...)'),
+            ('msisdn_za', 'South African contact number (+27)'),
+            ('msisdn_ke', 'Kenyan contact number (+254)'),
+            ('msisdn_ug', 'Ugandan contact number (+256)'),
+            ('msisdn_gh', 'Ghanaian contact number (+233)'),
+            ('msisdn_cm', 'Cameroonian contact number (+237)'),
+            ('msisdn_ng', 'Nigerian contact number (+234)'),
```
Comment (hodgestar) [inline go/contacts/parsers/base.py:35]: Do we maybe want a generic msisdn one too for internationalized numbers?

Comment (smn) [inline go/contacts/parsers/base.py:35]: I'm not sure what that does and why the `Leave as is` option wouldn't work for that?

File: go/contacts/tests.py [hunk 2]
```diff
@@ -283,6 +297,9 @@ def test_contact_parsing_failure(self):
             'column-0': 'name',
             'column-1': 'surname',
             'column-2': 'msisdn',
+            'normalize-0': '',
+            'normalize-1': '',
```
Comment (jerith) [inline go/contacts/tests.py:301]: Do you need this assignment if you're not using the value anywhere? pyflakes will complain...

Comment (smn) [inline go/contacts/tests.py:301]: Why will pyflakes compain? With sending a blank string as a dict value I'm telling the backend that no normalization should take place and it leaves the value as is.

Comment (jerith) [inline go/contacts/tests.py:301]: Um. I must have been commenting on an outdated diff. The line I commented on was `response = <thing that was there before>`, which seems to be gone now. Sorry about that.

**Comment by jerith** [comment]
If we're not doing any _validation_ on the normalized fields, we should probably be a little more explicit about that. Definitely in the code (docstring or comment or something), and probably a little note in the UI as well.

File: go/contacts/tests.py (hunk 2 above)
Comment (smn) [inline go/contacts/tests.py:301]: Yeah removed that late last night.

**Comment by jerith** [comment]
Perhaps we shouldn't push things through strings and then turn them into unicode afterwards? At the very least, we should probably specify character encodings when we do conversions.

**Comment by jerith** [comment]
:+1:

**Pull request merged**